2101. Detonate the Maximum Bombs/solution.py

```diff
@@ -0,0 +1,13 @@
+class Solution:
+    def maximumDetonation(self, bombs: List[List[int]]) -> int:
+        ### Creating the Bomb Graph ###
+
+        # initialize graph, a graph of bombs
```

Comment:
Graph is not a valid Python data structure so this comment is confusing especially since you already have a comment saying this section is about creating the Bomb Graph.